cdc/sink/producer/kafka/kafka.go

```diff
@@ -428,43 +445,81 @@ func validateMaxMessageBytesAndCreateTopic(admin kafka.ClusterAdminClient, topic
 	return nil
 }
 
-func getBrokerMessageMaxBytes(admin kafka.ClusterAdminClient) (int, error) {
+func validateMinInsyncReplicas(admin kafka.ClusterAdminClient,
+	topics map[string]sarama.TopicDetail, topic string, replicationFactor int) error {
+	info, exists := topics[topic]
+	if exists {
+		minInsyncReplicasStr, err := getTopicConfig(admin, info,
+			kafka.MinInsyncReplicasConfigName,
+			kafka.MinInsyncReplicasConfigName)
+		if err != nil {
+			return err
+		}
+		minInsyncReplicas, err := strconv.Atoi(minInsyncReplicasStr)
+		if err != nil {
+			return err
```

Comment:
Better to use an rfc error here.

Reply:
Warped in https://github.com/pingcap/tiflow/blob/4b4f59676601b25d8c17fbb8564c7d781b2d779f/cdc/sink/producer/kafka/kafka.go#L357